### ISSUE TITLE
More details about training

### ISSUE THREAD

**Issue by Sanster** [gwxie/Document-Dewarping-with-Control-Points#12]
Thanks for sharing the code and dataset. The encoder-only architecture makes DDCP faster and lighter than other methods, I really like the idea. I try to reimplement the paper, however, some training details are missing in the paper.

**loss**
<img width="713" alt="image" src="https://user-images.githubusercontent.com/3998421/182282715-331a2bd9-9191-412d-bf72-3a3b5d5451cf.png">

1. What are `α` and `β` used for the pre-train model? In the [utilsV4.py](https://github.com/gwxie/Document-Dewarping-with-Control-Points/blob/main/Source/utilsV4.py#L267) it's all equal `1`

**Experiments**
<img width="719" alt="image" src="https://user-images.githubusercontent.com/3998421/182282910-8c3d94ef-2544-4d5a-ad78-4b3b327be893.png">

2. What are the total epochs for the pre-train model? In [train.py](https://github.com/gwxie/Document-Dewarping-with-Control-Points/blob/main/Source/train.py#L263), the default `epochs=300`

**Comment by gwxie** [comment]
> 
Hi,
1、please see here.https://github.com/gwxie/Document-Dewarping-with-Control-Points/blob/ba4639ff1a0c8884d549e7fae3a9bddf1bef14ab/Source/train.py#L116
2、We have printed the EPOCH of  pre-train model. see here
https://github.com/gwxie/Document-Dewarping-with-Control-Points/blob/ba4639ff1a0c8884d549e7fae3a9bddf1bef14ab/Source/train.py#L93

**Comment by Sanster** [comment]
Thanks for your response! 

Have you tried adding a semantic segmentation head?  I tried to add an encoder to predict document mask, but the network does not converge.

<img width="1263" alt="image" src="https://user-images.githubusercontent.com/3998421/184781064-550af2e2-6a8b-4029-bf0d-548361e5d4f1.png">

![image](https://user-images.githubusercontent.com/3998421/184781420-28856f0b-da78-44ac-b5ea-533c963020f7.png)

**Comment by gwxie** [comment]
> Thanks for your response!
> 
> Have you tried adding a semantic segmentation head? I tried to add an encoder to predict document mask, but the network does not converge.
> 
> <img alt="image" width="1263" src="https://user-images.githubusercontent.com/3998421/184781064-550af2e2-6a8b-4029-bf0d-548361e5d4f1.png">
> 
> ![image](https://user-images.githubusercontent.com/3998421/184781420-28856f0b-da78-44ac-b5ea-533c963020f7.png)

Hi，
I've never done anything like this before.
